contrib/pg_tde/log/postmaster.log

```diff
@@ -0,0 +1,1676 @@
+2025-02-12 21:07:26.308 WET postmaster[1491794] LOG:  Initializing TDE principal key info
```

Comment:
Do we need this and log/initdb.log files? Looks like leftovers

Reply:
No, this was generated by some of the test runs, I'll add a gitignore